### PR TITLE
include silverstripe core files into roots

### PR DESCRIPTION
**Pull request by lekoala**
Fixes https://github.com/silverstripe/silverstripe-framework/issues/10892

### CODE REVIEW

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Could you add a unit test to cover the scenario here. Could be as simple as testing that Email sub classes are being returned now.

**Comment by lekoala** [comment]
@maxime-rainville hi maxime, so i did try to add a quick test but it's a bit more complicated than expected and i'd like your feedback and how you would like things done

- i've tried to add the assertion to ClassInfoTest::testSubclassesFor but that doesn't work since there is no proper manifest being built here (so it's empty). The current test only works because it's working with manually registered dataobjects. And its seems adding a whole manifest in the test just for that assertion would be a lot of code.

an alternative would be to:

- add to the emailtest, but again there are no manifest and that would means setting up a whole manifest in the test just for that
- or add the test to the ClassManifestTest and add new fixtures, but again, that seems a bit sad since we already have a email subclass in the tests

File: tests/php/Core/ClassInfoTest.php [hunk 1]
```diff
@@ -87,6 +87,14 @@ public function testSubclassesFor()
             $subclassesWithoutBase,
             ClassInfo::subclassesFor('silverstripe\\core\\tests\\classinfotest\\baseclass', false)
         );
+
+        // Check that core classes are present (eg: Email subclasses)
+        $emailClasses = ClassInfo::subclassesFor(SilverStripe\Control\Email\Email::class);
```
Comment (maxime-rainville) [inline tests/php/Core/ClassInfoTest.php]: Try this.

```suggestion
        $emailClasses = ClassInfo::subclassesFor(\SilverStripe\Control\Email\Email::class);
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]

File: tests/php/Core/ClassInfoTest.php (hunk 1 above)
Comment (lekoala) [reply tests/php/Core/ClassInfoTest.php]: @maxime-rainville now i feel stupid to have missed that :-) done !

**Comment by maxime-rainville** [comment]
I just rebased the PR to squsah all the commits into one. Also the build hadn't run in that last commit for some reason. So that should re-trigger the workflow.

**Review by maxime-rainville — APPROVED** [review]
Merged on green.

**Pull request merged**